rfd/0035-aws-node-join.md

```diff
@@ -0,0 +1,294 @@
+---
+authors: Nic Klaassen (nic@goteleport.com)
+state: draft
+---
+
+# RFD 35 - Simplified Node Joining for AWS
+
+## What
+
+EC2 instances will be able to join a teleport cluster without explicitly
+provisioning a join token on the auth server.
+
+In place of a join token, nodes will present one or more signed AWS API
+requests to the auth server. The auth server will send these requests to the
```

Comment:
Perhaps a quick definition of Signed Requests would be helpful. Maybe even just a link to [something like this](https://docs.aws.amazon.com/general/latest/gr/signing_aws_api_requests.html) will do